clang/test/Driver/linker-wrapper-sycl.cpp

```diff
@@ -0,0 +1,16 @@
+/// Check for no crashes for SYCL compilation using new offload model
+/// TODO(NOM4): Enable when driver support is available
+// RUNXXX: %clangxx -fsycl --offload-new-driver %s
+
```

Comment:
This test is supposed to validate the clang-linker-wrapper tool behavior, so we should move this check from this file.

Reply:
I will remove this chunk and will add this check one the driver changes are in.

Thanks